getting-started/getting-help.rst

```diff
@@ -13,40 +13,53 @@ Should you require help, there are a :ref:`variety of options available
 usage then please check the rest of this guide first as it should answer your
 question.
 
+
+.. _help-discourse:
+
 Discourse
 ---------
 
-Python has a hosted :ref:`Discourse <discourse_discuss>` instance. This forum has many different
-categories and most core development discussions take place in the open forum
-categories for `PEPs`_ and `Core Development`_ .
-Most categories are open for all users to read and post with the exception of
-Committers and Core Development categories. Be sure to visit the related Core
-categories, such as
-`Core Development <https://discuss.python.org/c/core-dev/23>`_ and
-`Core Workflow <https://discuss.python.org/c/core-workflow/8>`_.
+Python has a hosted Discourse instance at `discuss.python.org <Discourse_>`_.
```

Comment:
Can't you use `discuss.python.org` as link target below, instead of going through `Discourse_`?

IOW:
```rst
Python has a hosted Discourse instance at `discuss.python.org`_.

[...]

.. _discuss.python.org: https://discuss.python.org/

Reply:
I mean, I guess I _could_, but why repeat ourselves and go to that effort when we could just use an existing target label?